docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -232,6 +232,28 @@ To manage the media controller features in your application, you must learn to s
 
       The `watcherId` variable stores the value, which can be used in the future to remove the listener from the server using the `removeCommandListener()` method.
 
+
```

Comment:
Just asking your opinion.
From line 12 of this page, there is a list that showing main features of "Media Controller API".
(Setting up client and server pair, Managing request, ...etc)
Don't we need to add this "Setting content age rating" at that list, too?

Reply:
You are right, i have just added it.